Userland/Libraries/LibUnicode/CharacterTypes.cpp

```diff
@@ -222,4 +222,72 @@ bool code_point_has_property([[maybe_unused]] u32 code_point, [[maybe_unused]] P
 #endif
 }
 
+bool is_ecma262_property([[maybe_unused]] Property property)
+{
+#if ENABLE_UNICODE_DATA
+    // EMCA-262 only allows a subset of Unicode properties. The commented out properties are not yet parsed by LibUnicode.
```

Comment:
Oh, nevermind, I misunderstood the comments: These are _not_ parsed by the generator, and they don't exist in the list of enums.

Hmm. In this case, I think it would be good to clarify the comment a bit more.
```
// Some of these properties are not in the Property enum, as they are not yet parsed by the LibUnicode generator. They are left as commented-out until they can be handled properly.
```

Reply:
> they don't exist in the list of enums.

Exactly :) I updated the comment